site/content/docs/11-deploying.md

```diff
@@ -4,6 +4,16 @@ title: Deployment
 
 Sapper apps run anywhere that supports Node 8 or higher.
```

Comment:
```suggestion
Sapper apps run anywhere that supports Node 12 or higher.
```

According to https://github.com/sveltejs/sapper-template/issues/206#issuecomment-596485705, Node 8 support has been dropped. Should this be changed to the latest LTS version too?

Reply:
I think saying "Node 10 or higher" here would be most accurate